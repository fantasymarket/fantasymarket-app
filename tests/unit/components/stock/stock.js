import React from 'react';
import { render } from '@testing-library/react';

import Stock from 'components/stock';

describe('Stock component', () => {
	const container = render(<Stock />);

	it('renders the component', () => {
		expect(container.firstChild).toMatchSnapshot();
	});
});
