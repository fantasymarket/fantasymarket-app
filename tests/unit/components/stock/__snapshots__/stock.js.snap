// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stock component renders the component 1`] = `undefined`;
